content/800-data-platform/050-data-proxy/index.mdx

```diff
@@ -0,0 +1,101 @@
+
+A Data Proxy connection pool maintains active database connections and ensures that an application reuses them when new requests come in.
+
+In each region, Data Proxy management services orchestrate the lifecycle of connection pools. The lifecycle of a Data Proxy connection pool goes through the following stages:
+
+* the assignment of connection pools from each available cluster in the region
+* the increase of connection pools when the workload of your application increases
+* the maintenance of idle connection pools
+* the decrease and stop of connection pools after six minutes of inactivity in a pool
+
+This orchestration ensures that you accumulate Data Proxy usage (in CPU milliseconds) only when your database processes queries.
+
+The orchestration of Data Proxy connection pools ensures high availability, load balancing, and scalability of your application.
+
+* **High availability**. In each region, the Data Proxy infrastructure runs multiple clusters. By default, the Data Proxy starts a connection pool from each cluster and assigns each connection pool to your application. This remains valid until your application reaches higher workloads when the Data Proxy adds new connection pools as needed. If any of the clusters go through a period of downtime, the remaining ones take over the management of connection pools.
+    <Admonition>
+
+    **Note**<br /><br />
+
+    You can select only one of the available regions for an environment of your application in the Prisma Data Platform.<br /><br />
+
+    Because of this, the high availability of Data Proxy infrastructure applies inside a single region. Data Proxy connection pools from other regions cannot take over an application's connection pool from the current region.
+
+    </Admonition>
+* **Load balancing**. In each region, the number of incoming requests for an application is distributed between multiple connection pools running from the different clusters available in the region.
+* **Scalability**. Data Proxy creates new connection pools during periods of higher activity and closes connection pools when they are no longer necessary. This orchestration provides the flexibility to scale your database application and allows for reduced costs at times of lower activity.
+
+## Reduced bundle size
+
+By default, Prisma Client bundles the query engine that includes the connection logic for all databases that Prisma supports. 
+
+<img
+    src="https://website-v9.vercel.app/data-platform/without-data-proxy.svg"
+    alt="Prisma Client without the Data Proxy"
+    width="700px"
+/>
+
+Data Proxy includes the query engine logic to connect and maintain connection pools. This means that when you generate Prisma Client to work with Data Proxy, your application bundle does not need to include the query engine logic. As a result, your application has a reduced bundle size, which speeds up load times and improves performance.
```

Comment:
This actually also improves the cold start time, as the serverless function need less time to start and be ready to use.